apps/rush-lib/src/cli/logic/InstallManager.ts

```diff
@@ -157,47 +157,47 @@ export default class InstallManager {
     const packageManager: PackageManager = this._rushConfiguration.packageManager;
     const packageManagerVersion: string = this._rushConfiguration.packageManagerToolVersion;
 
+    const packageManagerAndVersion: string = `${packageManager}-${packageManagerVersion}`;
     // Example: "C:\Users\YourName\.rush\pnpm-1.2.3"
-    const packageManagerToolFolder: string = path.join(rushHomeFolder,
-      `${packageManager}-${packageManagerVersion}`);
-
-    // Example: "C:\Users\YourName\.rush\pnpm-1.2.3\last-install.flag"
-    const packageManagerToolFlagFile: string = path.join(packageManagerToolFolder, 'last-install.flag');
-
-    // NOTE: We don't care about the timestamp for last-install.flag, because nobody will change
-    // the package.json for this case
-    if (forceReinstall || !fsx.existsSync(packageManagerToolFlagFile)) {
-      console.log(colors.bold(`Installing ${packageManager} version ${packageManagerVersion}${os.EOL}`));
-
-      Utilities.installPackageInDirectory(
-        packageManagerToolFolder,
-        packageManager,
-        this._rushConfiguration.packageManagerToolVersion,
-        `${packageManager}-local-install`,
-        MAX_INSTALL_ATTEMPTS
-      );
-
-      // Create the marker file to indicate a successful install
-      fsx.writeFileSync(packageManagerToolFlagFile, '');
-      console.log(`Successfully installed ${packageManager} version ${packageManagerVersion}`);
-    } else {
-      console.log(`Found ${packageManager} version ${packageManagerVersion} in ${packageManagerToolFolder}`);
-    }
+    const packageManagerToolFolder: string = path.join(rushHomeFolder, packageManagerAndVersion);
 
-    // Example: "C:\MyRepo\common\temp"
-    if (!fsx.existsSync(this._rushConfiguration.commonTempFolder)) {
-      fsx.mkdirsSync(this._rushConfiguration.commonTempFolder);
-    }
+    console.log(`Trying to acquire lock for ${packageManagerAndVersion}`);
```

Comment:
> console.log(`Tr [](start = 3, length = 16)

Does this get printed on every run?  It might be a little noisy

Reply:
Not any noisier than the package manager. 

---
In reply to: [168682735](https://github.com/Microsoft/web-build-tools/pull/529#discussion_r168682735) [](ancestors = 168682735)

Reply:
Plus it is important for the user to know we are attempting to get the lock.

---
In reply to: [168846399](https://github.com/Microsoft/web-build-tools/pull/529#discussion_r168846399) [](ancestors = 168846399,168682735)